tpm2/tpm2.go

```diff
@@ -1368,3 +1368,116 @@ func ReadPCR(rw io.ReadWriter, pcr int, hashAlg Algorithm) ([]byte, error) {
 	}
 	return pcrVal, nil
 }
+
+// EncryptSymmetric encrypts data using a symmetric key. iv is the
```

Comment:
nit: Please reword to avoid having a lower-case code identifier as the first word in a sentence. E.g., this could be "The iv argument may be ..."

Reply:
Fixed.